assets/sass/_hero.scss

```diff
@@ -0,0 +1,61 @@
+header {
+
+    &.header-hero {
+        @apply bg-purple-500 text-white pt-8 pb-24 px-4;
+
+        @screen lg {
+            @apply py-24;
+        }
+
+        h1 {
+            @apply text-white;
+        }
+
+        > div {
```

Comment:
I'm going to add some comments here, and give hero items class some classnames, just for clarity.